content/README.md

```diff
@@ -26,4 +26,4 @@ When the module is compiled (using `dotnet build`) it's configured to generate a
 
 ## Notes
 
-This module was created using `v1.3.0` of [Etch.OrchardCore.ModuleBoilerplate](https://github.com/EtchUK/Etch.OrchardCore.ModuleBoilerplate) template.
+This module was created using `v1.4.0` of [Etch.OrchardCore.ModuleBoilerplate](https://github.com/EtchUK/Etch.OrchardCore.ModuleBoilerplate) template.
```

Comment:
should I change this to $(VersionNumber) inside the quotes, and then add the README to the targetted files? 

will have to make it specific, as we have two README's but yeah let me know